tests/test_handle_event.py

```diff
@@ -1,12 +1,15 @@
 import pytest
-from src.event_handler import EventHandler
+
+from src.event_handler import EventHandler, valid_payload
+from .fixtures import *
```

Comment:
eheh we should set up some linters -- or move the fixtures to `conftest.py`.  making fixtures available by import side-effects is ~technically not a supported mechanism in pytest (they should either be in a `conftest` or a pytest plugin) and only kinda work by accident right now

Reply:
Isn't it more fun to walk off the beaten path? :) I will look into it